src/io/parser/sparql.rs

```diff
@@ -28,22 +28,6 @@ pub enum Name<'a> {
     BlankNode(&'a str),
 }
 
-impl<'a> Name<'a> {
-    pub fn as_iri_reference(&'a self) -> Option<&'a str> {
-        match *self {
-            Name::IriReference(iri) => Some(iri),
-            _ => None,
-        }
-    }
-
-    pub fn as_blank_node_label(&'a self) -> Option<&'a str> {
-        match *self {
-            Name::BlankNode(label) => Some(label),
-            _ => None,
-        }
-    }
-}
-
```

Comment:
For what it's worth, my local `clippy` (nor `rustc`) doesn't complain about this either. Even more puzzling, since `dead_code` is not a clippy lint at all, but rather a compiler warning. Since the items are `pub`, they also shouldn't be considered dead anyways.

Reply:
Ah yes, it is indeed a compiler warning, so I also get this when just running `cargo b`.
But indeed, I would also assume the methods not `dead` because they are `pub`...

I'm currently on 
```
rustc 1.71.0-nightly (458d4dae8 2023-04-25)
cargo 1.71.0-nightly (de80432f0 2023-04-21)
```